src/locales/default.json

```diff
@@ -63,8 +63,8 @@
   "seeInExplorer": "See on explorer",
   "learnMore": "Learn more",
   "logout": "Log out",
-  "walletNotice": "Wallet not supported",
-  "gnosisSafeWalletNotice": "You are trying to use Gnosis Safe wallet, which doesn't support signing messages on Snapshot.",
+  "walletNotice": "Wallet notice",
+  "gnosisSafeWalletNotice": "Gnosis Safe wallet doesn't support signing messages, you can delegate your voting power on our delegation page.",
```

Comment:
Gnosis Safe wallet doesn't support signing messages. You can delegate your voting power to a wallet of your choice on our delegation page.